models/ncNews.model.js

```diff
@@ -0,0 +1,33 @@
+const db = require("../db/connection");
+
+exports.fetchAllTopics = () => {
+  return db
+  .query(
+    `
+      SELECT * FROM topics;
+  `
+  )
+  .then((result) => {
+    return result.rows;
+  })
+};
+
+exports.fetchArticleById = (article_id) => {
+  if (!Number.isInteger(parseInt(article_id))) {
```

Comment:
We may be able to reply on psql to give us an error rather than handling this manually here, see if we get an error code back that we could catch and pass to an error handler